docs/source/models/supported_models.rst

```diff
@@ -161,6 +161,10 @@ Decoder-only Language Models
     - Xverse
     - :code:`xverse/XVERSE-7B-Chat`, :code:`xverse/XVERSE-13B-Chat`, :code:`xverse/XVERSE-65B-Chat`, etc.
     -
+  * - :code:`DeepSeekMultiModalityCausalLM`
```

Comment:
We have added a new list to this page specifically for VLMs. Please move your entry to the new list.

Reply:
OK，It has been modified